### DISCUSSION TITLE
3.0.9

### DISCUSSION THREAD

**Discussion by timmo001** [timmo001/system-bridge#1695] [General]
## 🐛 Bug fixes

- Fix extra slashes on disk @timmo001 (#1692)

## 🚀 Enhancements

- Convert list string to list @timmo001 (#1688)
- Output disk partitions as list @timmo001 (#1689)
- Disk device list @timmo001 (#1690)
- Sort dictionary @timmo001 (#1691)
- Display keys to list @timmo001 (#1693)
- GPU keys to list @timmo001 (#1694)

<hr /><em>This discussion was created from the release <a href='https://github.com/timmo001/system-bridge/releases/tag/3.0.9'>3.0.9</a>.</em>

**Comment by ShurikenGitHub** [comment]
Could be my incompetence :sweat_smile:, but why is there no system-bridge-setup.exe file in the release assets? Version 2.8.4 was the last one with that asset. Would be very nice if you could re-add those to the releases so I don't have to compile manually every release.

**Reply by timmo001** [reply]
Please check the release notes for [3.0.0](https://github.com/timmo001/system-bridge/releases/tag/3.0.0). The app has been rewritten and now has a new install procedure.
